main.go

```diff
@@ -26,9 +26,10 @@ import (
 	"strings"
 	"syscall"
 
-	"github.com/GoogleCloudPlatform/berglas/pkg/berglas"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/berglas/pkg/berglas"
```

Comment:
Our editors apparently disagree about how to organize imports 😄

Reply:
Ah I added `github.com/GoogleCloudPlatform/berglas` to local imports to separate local imports into a separate block. I can revert it though.